### DISCUSSION TITLE
How to obtain a xkpwd URL to load a given preset?

### DISCUSSION THREAD

**Discussion by sandrock** [bartificer/xkpasswd-js#92] [Q&A] [answered]
I am writing procedures and I would like to add some links that would load a specified preset. 

Something like `https://beta.xkpasswd.net/?preset=NTLM`

I can't find a way so I believe this is not implemented?

**Comment by bbusschots** [comment]
This is an excellent feature suggestion — can't promise a specific timeline, but it's on the list.

**Reply by hepabolu** [reply]
To be more specific: issue #93

**Comment by hepabolu** [comment, marked as answer]
@sandrock Do check out the current version of XKPasswd. @irsheep was so kind to create a pull request that implemented this feature. Thanks @irsheep. I'll close this issue. Any errors or remarks, please create a new issue.
